.github/workflows/codeql.yml

```diff
@@ -17,16 +17,15 @@ jobs:
       actions: read
       contents: read
       security-events: write
-    container: ${{ github.repository }}-build:unstable
+    container: registry.community.greenbone.net/community/gvm-libs:edge
 
     strategy:
       fail-fast: false
       matrix:
         language: [ 'c' ]
-        
+
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v4
+    - uses: .github/actions/install-dependencies-action
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
```

Comment:
can you change github/codeql-action/init@v3 and github/codeql-action/analyze@v3 ?